packages/api/test/e2e/nft.e2e.ts

```diff
@@ -535,4 +535,14 @@ describe('NFTs', () => {
 
   });
 
+
+  it('Find tokens listing on Azalea', async done => {
+    const api = await Api.create({network: 'azalea'});
+    const tokenInfo = await api.derive.nft.openCollectionListings('55');
+    console.log('token listing::',tokenInfo);
+    expect(tokenInfo).toBeDefined();
+    await api.disconnect();
```

Comment:
Add more `expect` to to ensure the return `tokenInfo` has the correct shape

Reply:
I plan to remove this test completely as the listing will expire one day and this will not work.. we have other test covered for openCollectionListings...